src/logics/Logic.h

```diff
@@ -93,37 +51,33 @@ class Logic {
     Map<SymRef,bool,SymRefHash,Equal<SymRef> >      ites;
     Map<SRef,SymRef,SRefHash>                       sortToIte;
     Map<SRef,bool,SRefHash,Equal<SRef> >            ufsorts;
+    Map<SRef,PTRef,SRefHash,Equal<SRef>>            defaultValueForSort;
 
+    bool isKnownToUser(SymRef sr) const { return getSymName(sr)[0] != s_abstract_value_prefix[0]; }
```

Comment:
I am wondering if `s_abstract_value_prefix` should not be just a single `char` instead of `const char*`.

Reply:
I'm not sure.  In the end it probably matters little.